tools/analysis_tools/browse_dataset.py

```diff
@@ -78,6 +78,7 @@ def main():
             osp.basename(img_path),
             img,
```

Comment:
Why is img not a uint8  type?

Reply:
Not all transforms use uint8. E.g. PhotoMetricDistortion.

Reply:
OK